docs/src/pages/examples/rag/hybrid-vector-search.mdx

```diff
@@ -0,0 +1,86 @@
+---
+title: "Example: Hybrid Vector Search | RAG | Mastra Docs"
+description: Example of using metadata filters with PGVector to enhance vector search results in Mastra.
+---
+
+import { GithubLink } from "../../../components/github-link";
+
+# Hybrid Vector Search
+
+When you combine vector similarity search with metadata filters, you can create a hybrid search that is more precise and efficient.
+This approach combines:
+
+- Vector similarity search to find the most relevant documents
+- Metadata filters to refine the search results based on additional criteria
+
+This example demonstrates how to use hybrid vector search with Mastra and PGVector.
+
+## Overview
+
+The system implements filtered vector search using Mastra and PGVector. Here's what it does:
+
+1. Queries existing embeddings in PGVector with metadata filters
+2. Shows how to filter by different metadata fields
+3. Demonstrates combining vector similarity with metadata filtering
+
+> **Note**: To learn how to create and store embeddings in PGVector, see the [Insert Embeddings in PGVector](./insert-embedding-in-pgvector) guide.
```

Comment:
Shouldn't this be a sequel to the metadata extraction one?

Reply:
Added a reference to metadata extraction as well.